docs/core/tools/dotnet-restore.md

```diff
@@ -84,10 +84,14 @@ Restores dependencies and tools for the project in the current directory.
     
 Restores dependencies and tools for the `app1` project found in the given path.
 	
-`dotnet restore --f c:\packages\mypackages`
+`dotnet restore -f c:\packages\mypackages`
     
 Restores the dependencies and tools for the project in the current directory using the file path provided as the fallback source. 
+
+`dotnet restore -f c:\packages\mypackages -f c:\packages\myotherpackages`
 	
+Restores the dependencies and tools for the project in the current directory using the 2 file paths provided as the fallback sources. 
```

Comment:
I was a developer in a previous life :-)

But some of these suggestions come from Microsoft Style Guide so I just try to make sure we're following our conventions.